experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -57,6 +57,9 @@
 # importLG(name)
 -[${prefix}-${name}.${locale}.lg](${prefix}-${name}.${locale}.lg)
 
+# importLGFromLib(name)
+-[${prefix}-${name}.${locale}.lg](../library/${prefix}-${name}.${locale}.lg)
```

Comment:
>[${prefix}-${name}.${locale}.lg] [](start = 1, length = 32)

I think you need to merge with main.  I simplified the names and fixed a bug in merging where you were using the pretty name rather than the path.